### ISSUE TITLE
x-plane crash after tug attaches to plane

### ISSUE THREAD

**Issue by RegY82** [skiselkov/BetterPushbackC#343]
hiya i'm attaching the log, but the last lines were (it was a long flight before, this was supposed to be the second flight) - this was the only mention of betterpushback after the landing, as in the continuation
[Log.txt](https://github.com/skiselkov/BetterPushbackC/files/5258744/Log.txt)

2020-09-22 02:02:06 BetterPushback[acf_outline.c:200]: Error parsing acf file: property _wing/15/_Croot not found
4:43:55.188 E/SYS: MACIBM_alert: return_is_nan(flt[p0].ax_wrl
4:43:55.188 E/SYS: MACIBM_alert: 
4:43:55.188 E/SYS: MACIBM_alert: 
4:43:55.188 E/SYS: MACIBM_alert: 
4:43:55.188 E/SYS: MACIBM_alert: C:/jenkins/design-triggered/source_code/app/X-Plane-f/../../engine/fm/fm_systems_objects/fm_indi.cpp:90
--=={This application has crashed!}==--
(Art controls are modified.)

no idea what that's about, but it hasn't happened the first time since i updated to 11.50 (i didn't use betas), before itwas working fine.

**Comment by budgiesmith** [comment]
Yep, just had the same problem. Had some AI iarcraft in so will try to remove and see if that makes a difference. Also using Pilot2ATC.

0:37:34.622 I/ATC: Aircraft p=1 (Navy) started airborne as an overflight at -29.014822, 30.716084, 4267.200195
0:37:40.354 E/SYS: MACIBM_alert: return_is_nan(flt[p0].ax_wrl
0:37:40.354 E/SYS: MACIBM_alert: 
0:37:40.354 E/SYS: MACIBM_alert: 
0:37:40.354 E/SYS: MACIBM_alert: 
0:37:40.354 E/SYS: MACIBM_alert: C:/jenkins/design-triggered/source_code/app/X-Plane-f/../../engine/fm/fm_systems_objects/fm_indi.cpp:90
--=={This application has crashed!}==--
(Art controls are modified.)
